src/balancerd/src/lib.rs

```diff
@@ -178,6 +182,10 @@ impl BalancerService {
         let metrics = BalancerMetrics::new(&cfg);
         let mut configs = ConfigSet::default();
         configs = dyncfgs::all_dyncfgs(configs);
+        match tracing_config(&configs) {
+            Ok(parameters) => parameters.apply(&cfg.tracing_handle),
+            Err(e) => warn!("unable to update tracing: {e}"),
+        }
```

Comment:
TODO(jkosh44) plumb this to dyncfg sync.

Reply:
You probably don't even need it here if you have it in the sync, right?

Are you thinking of adding another callback to `sync_launchdarkly_to_configset` that is invoked on every sync?

Reply:
Yes that's exactly what I was thinking (and just did).